journalapp/scripts/load_api.py

```diff
@@ -0,0 +1,109 @@
+# coding=utf-8
+import os
+import requests
+
+from journalapp.models import Entry
+from sqlalchemy import engine_from_config
+from sqlalchemy.orm import scoped_session, sessionmaker
+from zope.sqlalchemy import ZopeTransactionExtension
+
+
+DBSession = scoped_session(sessionmaker(extension=ZopeTransactionExtension()))
+
+
+def load_with_key():
+    """
+    Opens a DB session
+    Collects and processes a JSON into a DB session
+    Closes the session
+    """
+    # Get API key
+    api_key = os.environ.get('API_KEY', None)
+
+    # Get DB Session
+    database_url = os.environ.get('DATABASE_URL', None)
+    print(database_url)
+    engine = engine_from_config({'sqlalchemy.url': database_url})
+    DBSession.configure(bind=engine)
+
+    json_payload = {}
```

Comment:
It would be better to have set the empty payload to be the same  data type as the actual payload would be.